ddapm_test_agent/trace_snapshot.py

```diff
@@ -299,6 +380,11 @@ def _ordered_span(s: Span) -> OrderedDictType[str, TopLevelSpanValue]:
     for k in sorted(set(s.keys()) - set(order)):
         d[k] = s[k]  # type: ignore
 
+    if "span_links" in d:
+        for link in d["span_links"]:
+            if "attributes" in link:
+                link["attributes"] = OrderedDict(sorted(link["attributes"].items(), key=operator.itemgetter(0)))
```

Comment:
Would the addition of `attributes` to the data be confusing to library developers who might think that the library was the one adding this data? 

Would it make more sense to store this data in another object to separate the concerns?

Reply:
I'm not sure what you mean, this just replaces the existing dict by an OrderedDict?

Reply:
ohhh this is my bad 🤦‍♂️! I didn't see that `attributes` was already a thing and we're just ordering it here.